frontend/src/components/Notice.tsx

```diff
@@ -22,7 +23,13 @@ function Notice({ children }: { readonly children?: React.ReactNode }) {
       </div>
       {/* TODO */}
       {/* eslint-disable-next-line jsx-a11y/no-static-element-interactions */}
-      <span className={styles.closeButton} onClick={() => setVisible(false)}>
+      <span
+        className={styles.closeButton}
+        onClick={() => {
+          setVisible(false);
+          Sentry.captureMessage('Course evals banner dismissed');
```

Comment:
could have this be generic and forever but ya seems good

Reply:
Nah, we want to recollect analytics for every banner, no?